### ISSUE TITLE
Reconsider Flatpak packages

### ISSUE THREAD

**Issue by p5** [loft-sh/devpod#1375]
**Is your feature request related to a problem?**  
Installing DevPod on other Linux distributions comes with challenges dealing with AppImage dependencies.  It would be nice to have packages provided by Loft.sh for Flatpaks on Flathub so installing DevPod is as easy as any other software.

**Which solution do you suggest?**  
I noticed you are now working on the Tauri v2 migration, which comes with a lot of improvements.  The one I'm most looking forward to are the [Flatpak](https://tauri.app/distribute/flatpak/) and [RPM](https://tauri.app/distribute/rpm/) support which should make providing these packages trivial.
Flatpak is distro-agnostic, and works across many distributions, so I'd suggest starting with this to provide the most value to users.

**Which alternative solutions exist?**  
I, along with the other [Universal Blue](https://github.com/ublue-os/) collaborators, have been maintaining Fedora COPR RPM packaging of DevPod for a couple years, though this has a ton of problems, especially when running with Nvidia GPUs.  I'm hoping the switch to Tauri v2, and Tauri's native support for this format, will resolve these issues.

**Additional context**  
This has previously been rejected since Tauri v1 didn't support these packaging formats, which is why I'm hoping we can give it another shot.  :crossed_fingers:

**Comment by pascalbreuninger** [comment]
Hey @p5, you're right, with tauri v2 we _should_ have a significantly easier way of distributing flatpak and RPMs. In the past we've shied away from supporting more than one linux package type because of the added support and maintenance overhead. 

It's amazing that you and other collaborators stepped up and created their own packages, thank you!

I'm going to leave this issue open until we've changed the pipeline

**Comment by castrojo** [comment]
Happy to run into ya'll at Kubecon! As (quickly) discussed, I'd be happy to help link you up with the right folks at Flathub to get you going as fast as we can. Let me know how I can help, thanks!

**Comment by p5** [comment]
I've scoped this issue down to only providing Flatpak packages hosted on Flathub.  
Flatpaks reduce the need for RPMs and other distribution-specific packaging, so it seems the most cost-effective format to look at first.

**Comment by pascalbreuninger** [comment]
@p5 Agree, we've been working on flatpak support pre tauri v2 already but stopped because tauri added it natively
@castrojo Thanks for chiming in :)

**Comment by tulilirockz** [comment]
Hello everyone! I have been working on a few flatpaks recently due to some limitations with [Achillobator](https://github.com/centos-workstation/achillobator) regarding SELinux and I believe I could help out with the development of the flatpak version. I've seen that you all dropped development on https://github.com/loft-sh/sh.loft.devpod/, but it seems like it doesnt require that many changes. If I fix that up and add some metadata and stuff, would there be any change we could publish this to [Flathub](https://flathub.org/)?

**Comment by the-spyke** [comment]
Not being on Flathub is like not being on Docker Hub. Please provide a Flatpak version.

**Comment by yodatak** [comment]
Hello for now its on flathub but it failed to build because of some issue ( thanks for team for make this appen )

https://github.com/flathub/sh.loft.devpod/issues/1

**Comment by AverageHelper** [comment]
not stale.

**Comment by p5** [comment]
Devpod is now available on Flathub, so closing this issue as completed.

**Comment by holmboe** [comment]
DevPod on Flathub is not mentioned on https://devpod.sh/docs/getting-started/install

**Comment by nneto** [comment]
Is the flatpak abandoned? :(

<img width="784" height="322" alt="Image" src="https://github.com/user-attachments/assets/aacb1fe4-1003-42df-847b-08bbe8780a62" />

<img width="232" height="148" alt="Image" src="https://github.com/user-attachments/assets/deae2883-a455-49ed-ad4d-131f67de678f" />

"Stopped receiving core updates"; "unverified"; no mention on docs; last version on flathub is 0.6.10 (from january) ...
